### PR TITLE
Download the endpoint.sh to local.

### PR DESCRIPTION
**Pull request by iShiBin**
So that it could be copied to the image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,10 +188,12 @@ Type following commands on a terminal to prepare a minimal project first:
 mkdir custom-fluentd
 cd custom-fluentd
 
-# Download default fluent.conf. This file will be copied to the new image.
-# VERSION is v1.4 or v0.12 like fluentd version and OS is alpine or debian.
-# Full example is https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/v1.4/debian-onbuild/fluent.conf
-curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS-onbuild/fluent.conf > fluent.conf
+# Download default fluent.conf, and the . This file will be copied to the new image.
```
Comment (ganmacs) [inline README.md]: What is ", and the"?

**Comment by ganmacs** [comment]
Can you please follow the DCO?

File: README.md (hunk 1 above)
Comment (iShiBin) [reply README.md]: The whole section should be like below. So please update the doc and I will close this PR.

```
# Download default fluent.conf and entrypoint.sh. This file will be copied to the new image.
# VERSION is v1.7 or v0.12 like fluentd version and OS is alpine or debian.
# Full example is https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/v1.7/debian/fluent.conf

curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS/fluent.conf > fluent.conf
curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS/entrypoint.sh > entrypoint.sh
```

**Comment by ganmacs** [comment]
This PR looks good.
Can you please follow the DCO? then I'll merge this.

**Comment by iShiBin** [comment]
> This PR looks good.
> Can you please follow the DCO? then I'll merge this.

I followed the DCO but still have the error "Some checks were not successful". How to solve it?

**Comment by ganmacs** [comment]
see https://github.com/fluent/fluentd-docker-image/pull/206/checks?check_run_id=422295074

**Comment by iShiBin** [comment]
> see https://github.com/fluent/fluentd-docker-image/pull/206/checks?check_run_id=422295074

I tried this but failed. Will close this one and raise another PR.

**Pull request closed without merging**